Ports/python-3.6/package.sh

```diff
@@ -9,3 +9,8 @@ installopts="-j$(nproc) build_all"
 files="https://www.python.org/ftp/python/3.6.0/Python-3.6.0.tar.xz Python-3.6.0.tar.xz"
 
 export CONFIG_SITE=$(pwd)/config.site
+
+if ! [ -x "$(command -v python3)" ]; then
```

Comment:
It doesn't just need Python 3 (that would be a reasonable requirement), it needs the same version you're cross-compiling, i.e. 3.6

Reply:
Why is that? Can it be avoided so you can compile it with any python3?

Reply:
Hi guys, sorry for long waiting, had lot of stuff to do during the week.
So I investigated some further time into that. I tried to allow a different python 3 version that is not the exact same as the one used to build and it turn's out that this is not working, because the some python modules cannot be found etc. The Makefile contains a lot of calls to the host python and uses that output further.

So my Idea here would be to check if the python3 has either the exact version (similar check as in configure.ac) or has been built by our Toolchain (therefore, the binary is in Toolchain/Local/bin path). 

To assure the toolchain contains the same python version as the port, the Toolchain/BuildPython.sh could source the package.sh script from python-3.6 Port to have exact the same download link and version of python to build. 

What do you think about that?